lib/taskana-core/src/test/java/pro/taskana/impl/ClassificationServiceImplTest.java

```diff
@@ -290,19 +291,38 @@ public void testGetClassificationWithInvalidNullKey() throws ClassificationNotFo
         }
     }
 
+    @Test(expected = InvalidArgumentException.class)
+    public void testGetClassificationWithInvalidNullId()
```

Comment:
this should probably be named like "testThrowExceptionIdClassificationIsCreatedWithAnExplicitId" or similar. What do you think?

Reply:
Yes, you are right your name is more descriptive.